cordova-lib/src/cordova/platform.js

```diff
@@ -158,6 +160,11 @@ function addHelper(cmd, hooksRunner, projectRoot, targets, opts) {
                         return installPluginsForNewPlatform(platform, projectRoot, cfg, opts);
                     }
                 }).then(function() {
+                    // Save platform@version into platforms.json. i.e: 'android@https://github.com/apache/cordova-android.git'
+                    // If no version was specified, save the edge version
+                    var versionToSave = version ? version : platforms[platform].version;
```

Comment:
just a nitpick - there is no need for ternary operator, `version || platforms[platform].version` will work the same way